docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+    instance or lose their liveliness.
+  * A time interval longer than the one established on the service_cleanup_delay has elapsed since the moment the
+    service detected that the two previous conditions were met.
+
+* **History kind**: Controls the kind of the :ref:`historyqospolicy` associated with the Durability Service fictitious
+  entities.
+* **History depth**: Controls the depth of the :ref:`historyqospolicy` associated with the Durability Service fictitious
+  entities.
+* **Max samples**: Controls the maximum number of samples of the :ref:`resourcelimitsqospolicy` associated with the
+  Durability Service fictitious entities.
+  This value must be higher than the maximum number of samples per instance.
+* **Max instances**: Controls the maximum number of instances of the :ref:`resourcelimitsqospolicy` associated with the
+  Durability Service fictitious entities.
+* **Max samples per instance**: Controls the maximum number of samples within an instance of the
+  :ref:`resourcelimitsqospolicy` associated with the Durability Service fictitious entities.
+  This value must be lower than the maximum number of samples.
+
+.. note::
+     This QoS Policy concerns to Topic and DataWriter entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entities.
+
+.. _entityfactoryqospolicy:
+
+EntityFactoryQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^
+
+This QoS Policy controls the behavior of an :ref:`dds_layer_core_entity` when it acts as a factory for other entities.
+As you know, the entities can be created either enabled or disabled, a fact that can be controlled using this QoS
+Policy.
+By default, all the entities are created enabled, but if you change the value of the `autoenable_created_entities` to
+false, the new entities will be created disabled.
+
+List of QoS Policy data members:
+
++-----------------------------+-------------------------------+-------------------+
+| Data Member Name            | Type                          | Default Value     |
++=============================+===============================+===================+
+| autoenable_created_entities | bool                          | true              |
++-----------------------------+-------------------------------+-------------------+
+
+.. note::
+     This QoS Policy concerns to DomainParticipantFactory (as factory for DomainParticipant), DomainParticipant (as factory for
+     Publisher, Subscriber and Topic), Publisher (as factory for DataWriter) and Subscriber (as factory for DataReader).
+     :raw-html:`<br />`
+     It can be changed after the creation of the entities, but it only affects those entities created after the change.
+
+Example
+"""""""
+
+C++
+***
+.. literalinclude:: ../../../../../code/DDSCodeTester.cpp
+   :language: c++
+   :dedent: 8
+   :start-after: //DDS_CHANGE_ENTITY_FACTORY_QOS_POLICY
+   :end-before: //!
+
+XML
+***
+This QoS Policy cannot be configured using XML by the moment.
```

Comment:
```suggestion
This QoS Policy cannot be configured using XML at the moment.
```

Reply:
The meaning of those sentences is not the same, and I think it is okay as it is.

Reply:
"by the moment" means something like 'por momentos'
Maybe 'for the moment' gives more sense of something that is planned to change?